pac4j-jwt/src/main/java/org/pac4j/jwt/credentials/authenticator/JwtAuthenticator.java

```diff
@@ -40,20 +40,30 @@
 
     protected final Logger logger = LoggerFactory.getLogger(getClass());
 
-    private String secret;
+    private String signingSecret;
+    private String encryptionSecret;
 
     public JwtAuthenticator() {}
 
-    public JwtAuthenticator(final String secret) {
-        this.secret = secret;
+    public JwtAuthenticator(final String signingSecret) {
+        this(signingSecret, signingSecret);
```

Comment:
My feeling is that we should output a warning to recommend using the new constructor which is safer with 2 secrets. What do you think?